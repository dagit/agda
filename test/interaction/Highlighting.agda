module Highlighting where

Set-one : Set₂
Set-one = Set₁

record R (A : Set) : Set-one where
  constructor con

  field X : Set

  F : Set → Set → Set
  F A B = B

  field P : F A X → Set

  Q : F A X → Set
  Q = Q

postulate P : _

open import Highlighting.M

data D (A : Set) : Set-one where
  d : let X = D in X A
