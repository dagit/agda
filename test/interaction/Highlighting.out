(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Highlighting (Highlighting.agda).\n" t)
(agda2-info-action "*Type-checking*" " Checking Highlighting.M (Highlighting/M.agda).\n" t)
(agda2-info-action "*Type-checking*" " Finished Highlighting.M.\n" t)
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-highlight-load-and-delete-action "
(agda2-info-action "*Type-checking*" "Finished Highlighting.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "Sort _5  [ at Highlighting.agda:19,15-16 ]\n_6 : Set _5  [ at Highlighting.agda:19,15-16 ]\n" nil)
((last . 1) . (agda2-goals-action '()))
